src/trace/trigger.ts

```diff
@@ -303,7 +303,7 @@ export function extractTriggerTags(event: any, context: Context) {
     try {
       eventSourceARN = parseEventSourceARN(eventSource, event, context);
     } catch (error) {
-      logError(`failed to extract ${eventSource} arn from the event`, { error });
+      logDebug(`failed to extract ${eventSource} arn from the event`, { error });
```

Comment:
BTW, is there any problem with the way we log `{ error }`? From both user reports, I didn't see any error being logged, which is probably the most valuable part of this log.

Reply:
Yeah this is actually super confusing because looking at the code all it seems like we're doing is returning a field in an object and somehow an error is being thrown. I've tried to replicate it [here](https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgMJysg3gKGf5EAVwFsB1ACwggBsBnALkNICNoBuHAXxxxiJAIwwAPYhkAcwhhK1egAoEGJuigBKJsRJtMuAsijSiUcUqgA6LbNp1OPHAjF0whEWFXIAvNns4wUAE9sPHwpGSobeRA3VTVuZCUwBApkeQg1YP1HEDoRGghzGhEJeQAiAFEADyQAB2ExZDAKKBEAdxBSuKynPIKikvT4mFA4GhogvQJs3PzC4rKAMQBJADkAQQAZDYBNTu5eIA).